### PR TITLE
Update lakeFS release to v0.103.0

### PR DESCRIPTION
**Pull request by N-o-Z**

### CODE REVIEW

File: charts/lakefs/README.md [hunk 1]
```diff
@@ -105,19 +112,19 @@ helm upgrade -f my-values.yaml my-lakefs lakefs/lakefs --set kv_upgrade=true
 
 ## Configurations
```
Comment (Jonathan-Rosenberg) [inline charts/lakefs/README.md:113]: Is the change below intentional?

Comment (N-o-Z) [reply charts/lakefs/README.md:113]: Yes - it's a fix of the table formatting

File: charts/lakefs/README.md [hunk 2]
```diff
@@ -75,6 +75,13 @@ If you can't provide such access, lakeFS can be configured to use an AWS key-pai
 
 ## Notable Chart Upgrades
 
+### Upgrading from chart version 0.9.4 or lower
+
+Introducing changes to the [security model in lakeFS](https://docs.lakefs.io/posts/security_update.html)  
+The lakeFS service will not run if the migration version isn't compatible with the binary.
+Before running the new version you will be required to run migrate, with the new version.
+Please refer to this [upgrade documentation](https://docs.lakefs.io/reference/access-control-lists.html#migrating-from-the-previous-version-of-acls) for more information on the specific migration from RBAC to ACL
```
Comment (Jonathan-Rosenberg) [inline charts/lakefs/README.md:83]:
```suggestion
Please refer to this [upgrade documentation](https://docs.lakefs.io/reference/access-control-lists.html#migrating-from-the-previous-version-of-acls) for more information on the specific migration from RBAC to ACL.
```
😉

**Review by Jonathan-Rosenberg — APPROVED** [review]

File: charts/lakefs/README.md (hunk 2 above)
Comment (N-o-Z) [reply charts/lakefs/README.md:83]: Sorry - PR was on auto-merge 😄

**Pull request merged**